lib/sinon/match.js

```diff
@@ -8,7 +8,6 @@
  */
 "use strict";
 
-var create = require("./util/core/create");
 var deepEqual = require("./util/core/deep-equal").use(match); // eslint-disable-line no-use-before-define
 var every = require("./util/core/every");
```

Comment:
Why can't we remove every use of `every` as well in favor of `Array#every`? Are they not doing the same thing?